src/server/models/db/migrations/20230713125822_move_list_item_audits_to_events/migration.sql

```diff
@@ -0,0 +1,16 @@
+-- Move Reminder audit events to Event table: (https://www.db-fiddle.com/f/pAsG9CgsiMz5YMZZo9FJ1t/0)
+-- NOTE: Event table entries contain response data from Notify emails. Since Audit doesn't have this, it won't be included when entries are moved
+INSERT INTO "Event" ("time", "listItemId", "type", "jsonData")
+SELECT  a."createdAt" "createdAt",
+        (a."jsonData"->>'itemId')::int,
+        "REMINDER",
```

Comment:
should be single quote, 'REMINDER'. 

Double quote is used when you want to refer to database identifiers, a bit like variables? "Event" - refers to a database name. "time" refers to a column. 

Single quotes are used for literals. i.e. use the literal value, 'REMINDER'.